src/Mimi.jl

```diff
@@ -121,13 +122,28 @@ end
 function setrandom(p::CertainArrayParameter)
 end
 
+type ComponentInstanceInfo
+  name::Symbol
+  component_type #the type here is a Type? or ComponentState or something else?
+end
+
+type ParameterVariableConnection
+  source_variable_name::Symbol
+  source_component_name::Symbol
+  target_parameter_name::Symbol
+  target_component_name::Symbol
+end
+
 type Model
     indices_counts::Dict{Symbol,Int}
     indices_values::Dict{Symbol,Vector{Any}}
     components::OrderedDict{Symbol,ComponentState}
     parameters_that_are_set::Set{UTF8String}
     parameters::Dict{Symbol,Parameter}
     numberType::DataType
+    #model_graph::ModelGraph
+    edges::Array{ParameterVariableConnection, 1}
```

Comment:
Can we use a more specific name here, e.g. ``connections`` or something like that?